prdoc/pr_4959.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: relax XcmFeeToAccount trait bound on AccountId
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR relaxes the trait bound on AccountId for the XcmFeeToAccount struct by introducing a new struct called `SendXcmFeeToAccount`.
+      The old one (`XcmFeeToAccount`) will be deprecated at January 2025.
+
+crates:
+  - name: staging-xcm-builder
+    bump: minor
```

Comment:
you should also do `minor` bumps for all runtimes where the new one is used

Reply:
Done! Also did the same for `pallet-xcm`